db/migrations/202408061737_add_boostagrams_and_use_json.go

```diff
@@ -0,0 +1,30 @@
+package migrations
+
+import (
+	_ "embed"
+
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+// This migration adds boostagram column to transactions
+var _202408061737_add_boostagrams_and_use_json = &gormigrate.Migration{
+	ID: "202408061737_add_boostagrams_and_use_json",
+	Migrate: func(db *gorm.DB) error {
+		err := db.Transaction(func(tx *gorm.DB) error {
+			return tx.Exec(`
+			ALTER TABLE transactions ADD COLUMN boostagram JSONB;
+			ALTER TABLE transactions ADD COLUMN metadata_temp JSONB;
+			UPDATE transactions SET metadata_temp = "";
+			UPDATE transactions SET metadata_temp = json(metadata) where metadata != "";
```

Comment:
a few things seem wrong here, it doesn't work with existing transactions, it uses the json() function but the column types are jsonb, I'm not sure what to do with existing metadata values that were an empty string - if they should be changed NULL in this case

No json queries are working for me.